images/build/Dockerfile

```diff
@@ -56,44 +56,84 @@ ENV DOTNET_RUNNING_IN_CONTAINER=true \
 	NUGET_PACKAGES=/var/nuget
 
 RUN mkdir /var/nuget
-COPY build/__dot-net-core-versions.sh /tmp
```

Comment:
FYI: This is how all lays out

![image](https://user-images.githubusercontent.com/6559784/58295098-4fd39e80-7d82-11e9-8dd6-659fce0672ca.png)